elixirkit/lib/elixirkit/server.ex

```diff
@@ -24,7 +24,7 @@ defmodule ElixirKit.Server do
 
   @impl true
   def handle_info({:tcp_closed, socket}, state) when socket == state.socket do
-    System.stop()
+    send(state.pid, :shutdown)
```

Comment:
Maybe don't send anything and tell the parent to monitor this process? Thoughts?

Reply:
The benefit of doing that is that we will also track crashes and everything else.

Reply:
Sounds good